itou/communications/models.py

```diff
@@ -0,0 +1,89 @@
+from django.conf import settings
+from django.contrib.contenttypes.fields import GenericForeignKey
+from django.contrib.contenttypes.models import ContentType
+from django.db import models
+
+
+class NotificationQuerySet(models.QuerySet):
+    def actives(self):
+        return self.exclude(is_obsolete=True)
+
+
+class Notification(models.Model):
+    notification_class = models.CharField(max_length=255, unique=True)
+    name = models.CharField(max_length=255)
+    category = models.CharField(max_length=255)
+    can_be_disabled = models.BooleanField(default=True)
```

Comment:
Je virerais bien le `default` pour obliger à un faire un choix, et vu qu'elles sont synchro au lancement de l'app ça peux éviter un oubli idiot.
```suggestion
    can_be_disabled = models.BooleanField()
```